NitroxModel/Packets/KnownTechEntry.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class KnownTechEntryAdd : Packet
```

Comment:
Please add all new packets to their own files (here and throughout the rest of the PR).  This has been the convention that we are following.